testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -50,8 +50,9 @@ const GENESIS_HELM_RELEASE_NAME: &str = "genesis";
 const APTOS_NODE_HELM_CHART_PATH: &str = "terraform/helm/aptos-node";
 const GENESIS_HELM_CHART_PATH: &str = "terraform/helm/genesis";
 
-// cleanup namespaces after 30 min unless "keep = true"
+// cleanup namespaces after 1 hour 30 minutes unless "keep = true"
```

Comment:
nit: change the comment to "30 minutes", and add a note that this field should be deprecated once the `cleanup` feature is fully rolled out